build/create_solr_index.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+
+SOLR_HOST=$(echo $SOLR_URL | cut -d '/' -f 1,2,3)
+
+# Upload the existing hyrax SOLR configurations from solr/config.zip
```

Comment:
There is a Solr trick that I've used for Avalon, mounting the `solr/config` folder and precreating core on startup (nothing happens if core already exists though). This way we won't have to maintain a separate, potentially out-of-sync `solr/config.zip` and simplify our setup.
```
  solr:
    image: solr:6.6.4-alpine
    volumes:
      - ./solr/config:/opt/solr/avalon_conf
      - solr:/opt/solr/server/solr/mycores
    entrypoint:
      - docker-entrypoint.sh
      - solr-precreate
      - avalon
      - /opt/solr/avalon_conf
```